src/main/java/aquarium/shop/product/Product.java

```diff
@@ -1,10 +1,15 @@
 package aquarium.shop.product;
 
 
+
 import javax.persistence.*;
 
+import java.util.Objects;
+
+import static javax.persistence.GenerationType.AUTO;
```

Comment:
Probably not needed import. In Intellij you can optimize your imports (removing unnecessary and changing orders if needed) by ctrl + alt + o.